.github/workflows/coverage-windows.yml

```diff
@@ -37,7 +37,7 @@ permissions:
 jobs:
   coverage-windows:
     if: github.event.pull_request.draft == false
-    runs-on: windows-2019
```

Comment:
I think this should stays on windows-2019 since this need a more stable build env. As my experience, MSVC stability can't be guaranteed and take way to many time to fix.